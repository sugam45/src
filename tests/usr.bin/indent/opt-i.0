/* $NetBSD: opt-i.0,v 1.2 2021/03/06 22:10:40 rillig Exp $ */
/* $FreeBSD$ */

void
example(void)
{
   if (1 > 0) if (2 > 1) return yes; return no;
}
