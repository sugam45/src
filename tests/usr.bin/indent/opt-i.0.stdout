/* $NetBSD: opt-i.0.stdout,v 1.2 2021/03/06 22:10:40 rillig Exp $ */
/* $FreeBSD$ */

void
example(void)
{
   if (1 > 0)
      if (2 > 1)
	 return yes;
   return no;
}
